handbook/product/product_management/direction_template.md

```diff
@@ -0,0 +1,100 @@
+
+---
+
+# MyProductArea Direction
+
+This page outlines the vision, strategy and goals of the MyProductArea team. You can use this section to describe what your scope is, and where your product is today.
+
+Quicklinks:
+
+- [MyProductArea Backlog](https://about.sourcegraph.com/) <!-- Update links with your page -->
+- [Latest demo](https://about.sourcegraph.com/)
+- [Documentation](https://about.sourcegraph.com/)
+
+# Mission & Strategy
+
+What is the team's reason for existing? What is the fundamental value they aim to provide?
+
+What is your strategy defined in a few steps?
+
+# Vision
+
+#### 1 Year vision
+
+> Add a sentence or two here indicating what the most important things you want to achieve/problems you want to solve in the next year are. These should in most cases move your three and ten year visions forward.
+
+#### 3 Year vision
+
+> Add a sentence or two here describing what your product will be like in three years, also focusing on the problems you'll solve and the value you'll deliver. This should be grounded in reality, but three years is a lot of time - be ambitious!
+
+#### 10 year vision
+
+> A sentence or two on your ten year vision is where you can really go wild and describe the problems you want to solve and solutions you want to deliver unbound by limitations of technology or scope. Be inspiring and look to the our [10 and 30 year strategies](../../company/strategy.md#sourcegraph-strategy) for ideas.
+
+# Guiding principles
+
+Identify what key principles, insights, and decisions are important to the strategy or a set of key hypotheses and set of key questions to validate or invalidate them.
+Describe the target by which we prioritize:
+
+- Market or audience
+- Personas
+- Jobs or Use Cases
+- Industries
+- What matters to us when we prioritize products and features
+- Contextual things that matter to Sourcegraph or the product
+
+# Where we are now
+
+- Where is the team's area of ownership in terms of its maturity? Is it new and basic, or complete and lovable? Are different features at different levels?
+- What did we achieve in the last few months?
+- What key learnings did we recently make?
+- What is on the critical path for growth?
+- How does the product fit within Sourcegraph as a whole?
+
+## Competitive landscape
+
+- Who is the competition for this product area?
+- What recent developments in competition is impacting sales, marketing and product strategy?
+
+## Analyst landscape
+
+- Are there analysts tracking this product area?
+- How are analysts positioning the product? What are areas of improvement?
+
+# What's next and why
+
+What are we going to do in the next quarter, and how does it accomplish company goals and the product vision?
+
```

Comment:
How about adding a subsection here that is specifically the Quarterly plans? 

That way we keep the longer term, loosely defined what's next under Themes, but we can also answer what the sales and CE team will be hoping to see here: what's coming up in the next quarter?